cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/OperationalLimitConversion.java

```diff
@@ -197,9 +197,13 @@ private void convertTatlCurrent(double value) {
         if (direction == null || direction.endsWith("high") || direction.endsWith("absoluteValue")) {
             if (currentLimitsAdder != null) {
                 addTatlCurrent(context.namingStrategy().getId("TATL", id), value, acceptableDuration, currentLimitsAdder);
-            } else if (currentLimitsAdder1 != null) {
-                // Should we chose one terminal randomly for branches ? Here by default, we only look at terminal1
-                addTatlCurrent(context.namingStrategy().getId("TATL", id), value, acceptableDuration, currentLimitsAdder1);
+            } else {
+                if (currentLimitsAdder1 != null) {
+                    addTatlCurrent(context.namingStrategy().getId("TATL", id), value, acceptableDuration, currentLimitsAdder1);
+                }
+                if (currentLimitsAdder2 != null) {
```

Comment:
Are you sure it’s an « if » not an « else if »?

Reply:
No, it is possible to have both non null adders when the CGMES `OperationalLimitSet` is linked to the equipment (not a terminal).